script/test_in_container/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# This Dockefile creates image where all Kompose tests can be run
+
+FROM registry.centos.org/centos/centos:7
+MAINTAINER Tomas Kral <tkral@redhat.com>
```

Comment:
Oh, I didn't know that.
 I'll remove it, I don't think that it is something that we need for image. It was just something that I was used to doing.